taskt/UI/CustomControls/CommandControls.cs

```diff
@@ -300,13 +300,13 @@ private static void ShowMouseCaptureForm(object sender, EventArgs e)
                 //Todo - ideally one function to add to textbox which adds to class
 
                 //add selected variables to associated control text
-                CurrentEditor.flw_InputVariables.Controls["v_XMousePosition"].Text = frmShowCursorPos.xPos.ToString();
-                CurrentEditor.flw_InputVariables.Controls["v_YMousePosition"].Text = frmShowCursorPos.yPos.ToString();
+                CurrentEditor.flw_InputVariables.Controls["v_XMousePosition"].Text = frmShowCursorPos.XPosition.ToString();
+                CurrentEditor.flw_InputVariables.Controls["v_YMousePosition"].Text = frmShowCursorPos.YPosition.ToString();
 
                 //find current command and add to underlying class
-                SendMouseMoveCommand cmd = (SendMouseMoveCommand)CurrentEditor.SelectedCommand;
-                cmd.v_XMousePosition = frmShowCursorPos.xPos.ToString();
-                cmd.v_YMousePosition = frmShowCursorPos.yPos.ToString();
+                SendMouseMoveCommand cmd = (SendMouseMoveCommand)CurrentEditor.selectedCommand;
```

Comment:
This should be the case that it was before, no? frmCommandEditor also has it cased as `SelectedCommand`.

Reply:
The new version is right but the selectedCommand was broken in the merge. I hadn't gotten to it because I hopped onto the fix